src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/ProportionalSumAggregator.java

```diff
@@ -172,6 +172,12 @@ public void collect(int doc, long bucket) throws IOException {
                     valueVal = valueDoubleValues.doubleValue();
                 }
 
+                // scale value by sampling interval
+                final NumericDoubleValues samplingDoubleValues = values[3];
+                if (samplingDoubleValues.advanceExact(doc)) {
+                    valueVal *= samplingDoubleValues.doubleValue();
```

Comment:
Should we avoid multiplying if the `sampling_interval` is 0? I ask this since the value in ES is currently 0 for Netflow v5 flows generated by softflowd.